Sources/libhostmgr/BuildkiteScriptBuilder.swift

```diff
@@ -13,7 +13,6 @@ public struct BuildkiteScriptBuilder {
     var commands = [Command]()
 
     public init() {
-        self.environmentVariables["BUILDKITE"] = Value(wrapping: "true")
     }
 
```

Comment:
Might as well remove the whole `init`?

```suggestion
```

Reply:
This init needs to be kept as a public API for the cli target to call.

Reply:
Ah right, forgot that the generated init for the structs were not public by default 👍